src/openzaak/components/zaken/api/viewsets.py

```diff
@@ -1424,20 +1427,46 @@ def perform_destroy(self, instance):
 
 @extend_schema_view(
     create=extend_schema(
-        "zaak_reserveer_zaaknummer",
+        operation_id="zaak_reserveer_zaaknummer",
         summary="Reserveer een zaaknummer",
         description=mark_experimental(
             "Reserveer een zaaknummer binnen een specifieke bronorganisatie zonder direct een Zaak aan te maken. "
             "Dit zaaknummer zal toegekend worden aan de eerstvolgende Zaak die met dit zaaknummer wordt aangemaakt "
-            "binnen de bronorganisatie en het zaaknummer kan daarna niet hergebruikt worden."
+            "binnen de bronorganisatie en het zaaknummer kan daarna niet hergebruikt worden.\n\n"
+            "Als `aantal` niet wordt opgegeven of gelijk is aan 1, wordt een enkel object teruggegeven.\n"
+            "Als `aantal > 1`, wordt een lijst van objecten teruggegeven."
         ),
         request=ReserveZaakIdentificatieSerializer,
         responses={
-            status.HTTP_201_CREATED: ReserveZaakIdentificatieSerializer(),
+            status.HTTP_201_CREATED: OpenApiResponse(
+                response=PolymorphicProxySerializer(
+                    component_name="OneOrMultipleZaakIdentificaties",
+                    serializers=[
+                        ReserveZaakIdentificatieSerializer,
+                        ReserveZaakIdentificatieSerializer(many=True),
+                    ],
+                    resource_type_field_name=None,
+                )
```

Comment:
Adding this will fix the linting errors for the OAS
```suggestion
                    resource_type_field_name=None,
                    many=False
                )
```